<ui version="4.0" >
 <class>AddressBarWidgetClass</class>
 <widget class="QWidget" name="AddressBarWidgetClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>296</width>
    <height>76</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>AddressBarWidget</string>
  </property>
  <property name="autoFillBackground" >
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <property name="leftMargin" >
    <number>2</number>
   </property>
   <property name="topMargin" >
    <number>2</number>
   </property>
   <property name="rightMargin" >
    <number>2</number>
   </property>
   <property name="bottomMargin" >
    <number>0</number>
   </property>
   <property name="spacing" >
    <number>1</number>
   </property>
   <item row="0" column="7" >
    <widget class="QPushButton" name="GoStopPushButton" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize" >
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize" >
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="text" >
      <string/>
     </property>
     <property name="icon" >
      <iconset>
       <normalon>:/images/go.png</normalon>
      </iconset>
     </property>
     <property name="iconSize" >
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="flat" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="3" colspan="4" >
    <widget class="QLineEdit" name="AddressLineEdit" >
     <property name="minimumSize" >
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize" >
      <size>
       <width>16777215</width>
       <height>40</height>
      </size>
     </property>
     <property name="frame" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="3" colspan="4" >
    <widget class="QProgressBar" name="LoadingProgressBar" >
     <property name="minimumSize" >
      <size>
       <width>0</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize" >
      <size>
       <width>16777215</width>
       <height>20</height>
      </size>
     </property>
     <property name="value" >
      <number>0</number>
     </property>
     <property name="alignment" >
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="textVisible" >
      <bool>false</bool>
     </property>
     <property name="invertedAppearance" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="7" >
    <widget class="QPushButton" name="SlidePushButton" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize" >
      <size>
       <width>40</width>
       <height>32</height>
      </size>
     </property>
     <property name="maximumSize" >
      <size>
       <width>40</width>
       <height>32</height>
      </size>
     </property>
     <property name="text" >
      <string/>
     </property>
     <property name="icon" >
      <iconset>
       <normalon>:/images/slide-vertical.png</normalon>
      </iconset>
     </property>
     <property name="iconSize" >
      <size>
       <width>40</width>
       <height>32</height>
      </size>
     </property>
     <property name="flat" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="7" >
    <spacer name="verticalSpacer" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="2" >
    <widget class="ChaseWidget" native="1" name="BusyLoadingChaseWidget" >
     <property name="enabled" >
      <bool>true</bool>
     </property>
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize" >
      <size>
       <width>40</width>
       <height>32</height>
      </size>
     </property>
     <property name="maximumSize" >
      <size>
       <width>40</width>
       <height>32</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="2" >
    <widget class="QPushButton" name="ReloadPushButton" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize" >
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize" >
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="text" >
      <string/>
     </property>
     <property name="icon" >
      <iconset>
       <normalon>:/images/reload.png</normalon>
      </iconset>
     </property>
     <property name="iconSize" >
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="flat" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <customwidgets>
  <customwidget>
   <class>ChaseWidget</class>
   <extends>QWidget</extends>
   <header>chasewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="images.qrc" />
 </resources>
 <connections/>
</ui>
